doc/conf.py

```diff
@@ -207,18 +208,19 @@
     "doc_path": "doc/",
 }
 
+html_additional_pages = {"test": "test.html"}
+
 # Python autodoc options
 autodoc_default_options = {
-    'member-order':    'bysource',
+    "member-order": "bysource",
 }
 
 
-
 def setup(self):
     import subprocess
     import pathlib
 
     srcdir = pathlib.Path(self.srcdir)
 
     breathe_projects["CCF"] = str(srcdir / breathe_projects["CCF"])
-    subprocess.run(["doxygen"], cwd=srcdir / '..', check=True)
+    # subprocess.run(["doxygen"], cwd=srcdir / '..', check=True)
```

Comment:
Revert before merging

Reply:
Why is it commented out?

Reply:
Running `doxygen` on every change makes the `./livehtml.sh` experience slow to get the newest version of the doc rendered in the browser.

Reply:
Ah ok